pubspec.yaml

```diff
@@ -12,7 +12,7 @@ dependencies:
   flutter:
     sdk: flutter
   yaru: ^0.9.0
-  yaru_icons: ^1.0.0
+  yaru_icons: ">=1.0.0"
```

Comment:
my bad, i didn't realize pub.dev being so cranky about this. what about ">=1.0.0 <3.0.0" to make pub.dev happy?